homeassistant/components/sensor/eliqonline.py

```diff
@@ -13,8 +13,9 @@
 from homeassistant.const import (CONF_ACCESS_TOKEN, CONF_NAME, STATE_UNKNOWN)
```

Comment:
It would be nice to clean up the use of `STATE_UNKNOWN` while we're here. Use `None` to represent unknown state.

Reply:
Ok